staging/vhost-device-gpu/README.md

```diff
@@ -64,7 +64,7 @@ First start the daemon on the host machine using either of the 2 gpu modes:
 2) `gfxstream` (if the crate has been compiled with the feature `gfxstream`)
 
 ```shell
-host# vhost-device-gpu --socket-path /tmp/gpu.socket --gpu-mode virgl-renderer
+host# vhost-device-gpu --socket-path /tmp/gpu.socket --gpu-mode virglrenderer
```

Comment:
Where is another occurrence in the readme of `virgl-renderer`: https://github.com/mtjhrc/vhost-device/blob/4787a30fc0ddd44cde2d09a5e8412bfb1573d275/staging/vhost-device-gpu/README.md?plain=1#L104